cdisc_rules_engine/utilities/rule_processor.py

```diff
@@ -262,7 +253,7 @@ def perform_rule_operations(
         return dataset_copy
 
     def _execute_operation(
-        self, operation_to_call: Callable, operation_params: OperationParams
+        self, operation_params: OperationParams, dataset: pd.DataFrame
```

Comment:
dataset exists in OperationParams

Reply:
The dataset in operation params is the dataset we are performing the operation on. More often than not, the dataset we are performing the operation on is different from the original dataset. So in order to always return a data frame from operation execute methods I opted to append the result to the original dataset

Reply:
Got it, then it is a good idea to rename the parameter to be like `original_dataset`